### PR TITLE
[performance] update references in Bandwidth/Latency doc

### PR DESCRIPTION
**Pull request by DavidScales**
Update **sw-precache** reference to **Workbox** in Performance > Loading Performance > Understanding Low Bandwidth and High Latency

**Fixes:** N/A

**Target Live Date:** 2018-01 (will need to be updated again when Workbox v3 arrives)

- [ ] This has been reviewed and approved by (NAME)
- [x] I have run `gulp test` locally and all tests pass.
- [x] I have added the appropriate `type-something` label.
- [x] I've staged the site and manually verified that my content displays correctly.

**CC:** @petele

### CODE REVIEW

File: src/content/en/fundamentals/performance/poor-connectivity/index.md [hunk 1]
```diff
@@ -4,6 +4,7 @@ description: It's important to understand what using your app or site feels like
 
 {# wf_updated_on: 2017-07-24 #}
```
Comment (petele) [inline src/content/en/fundamentals/performance/poor-connectivity/index.md]: Please update the `wf_updated_on` date

File: src/content/en/fundamentals/performance/poor-connectivity/index.md [hunk 2]
```diff
@@ -97,14 +98,13 @@ Lie-fi is likely to become a bigger problem as more people move to mobile and aw
 
 ### Use timeouts to handle intermittent connectivity
 
-In the past, [hacky methods using XHR](http://stackoverflow.com/questions/189430/detect-that-the-internet-connection-is-offline) have been used to test for intermittent connectivity, but service worker enables more reliable methods to set network timeouts. Jeff Posnick explains how to achieve this using [sw-toolbox](https://github.com/GoogleChrome/sw-toolbox) timeouts in his talk [Instant Loading with Service Workers](https://youtu.be/jCKZDTtUA2A?t=19m58s):
+In the past, [hacky methods using XHR](http://stackoverflow.com/questions/189430/detect-that-the-internet-connection-is-offline) have been used to test for intermittent connectivity, but service worker enables more reliable methods to set network timeouts. This can be achieved using [Workbox](/web/tools/workbox/) with only a few lines of code:
```
Comment (petele) [inline src/content/en/fundamentals/performance/poor-connectivity/index.md]: Please break this line at 80 chars, you don't have to do the rest of the file, but please do the ones you change.

**Comment by DavidScales** [comment]
whoops, sorry Pete! I've made the corrections. Note that I updated all of the offending lines to be less than 100 characters, not just the ones that I edited (assuming that's cool).

**Comment by petele** [comment]
that's SUPER awesome - thanks for breaking all those lines!

**Pull request merged**